utcp.c

```diff
@@ -26,9 +26,12 @@
 #include <stdint.h>
 #include <stdbool.h>
 #include <string.h>
+
+#ifndef _WIN32
 #include <unistd.h>
 #include <sys/time.h>
 #include <sys/socket.h>
+#endif
```

Comment:
richtig, finde ich aber nicht schlimm, dafür gibt's include guards und so sieht man nochmal lokal dass es included ist
soll ich unistd.h und sys/socket.h vll mit in compat.h aufnehmen?